tests/errors/test_set3.py

```diff
@@ -0,0 +1,5 @@
+def f():
+    s: set[i32]
+    s = {1, 2, 3, 4}
+    i: i32
+    i = s.pop(3)
```

Comment:
Can we test a use case like the following?

```python
>>> def return_set():
            return {1, 2, 3, 4}
>>> def f():
            i = return_set().pop()
           return i 
>>> f()
1
```

Reply:
See #501.  We have to make it work and register this as a test.